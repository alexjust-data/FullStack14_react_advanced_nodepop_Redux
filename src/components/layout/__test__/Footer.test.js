import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../Footer';

describe('Footer', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
