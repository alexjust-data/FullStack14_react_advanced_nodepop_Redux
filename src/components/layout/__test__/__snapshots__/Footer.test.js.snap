// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<DocumentFragment>
  <footer>
    @Keepcoding 2021
  </footer>
</DocumentFragment>
`;
